### ISSUE TITLE
[SUGGESTION] Antiafk that works on 2b2t

### ISSUE THREAD

**Issue by ghost** [pyroclient/pyro#225]
**Is your feature request related to a problem? Please describe.**
Add antiafk that works on 2b2t

**Describe the solution you'd like**
anti AFK module that keeps you from being kicked on 2b2t

**Comment by u3o** [comment]
Yeah, kami blue has it but kami crashes a lot and bad client overall so pls
